india_compliance/gst_india/utils/e_waybill.py

```diff
@@ -717,10 +717,12 @@ def mark_e_waybill_as_cancelled(doctype, docname, values):
 
 
 def attach_e_waybill_pdf(doc, log=None):
+    default_format = frappe.get_meta("e-Waybill Log").default_print_format
```

Comment:
this isn't required at all. if you don't specify any print format, this is what gets chosen.